.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
```

Comment:
Consdering this plugin can be used in the browser and in Node and both of this is tested in this workflow, I find the name to be a little misleading? Mochify just uses `Build` in a file called `test.yml`, maybe we just want to do the same thing here?